init.go

```diff
@@ -26,3 +33,18 @@ func Initialize(field *Field) error {
 
 	return nil
 }
+
+func InitializeEx(ctx context.Context, field *Field) error {
+	var err error
+	if field.Value.Type().Implements(typeInitializableEx) {
+		err = field.Value.Interface().(InitializableEx).Init(ctx)
+	} else if field.Value.Type().Implements(typeInitializable) {
```

Comment:
With the default processor, both the Initialize and InitializeEx hooks are used, so an Initializable element will be initialized twice

Reply:
Good catch